org.eclipse.jdt.ls.target/org.eclipse.jdt.ls.tp.target

```diff
@@ -19,14 +19,20 @@
             <unit id="org.eclipse.equinox.core.feature.feature.group" version="0.0.0"/>
             <unit id="org.eclipse.equinox.core.sdk.feature.group" version="0.0.0"/>
             <unit id="org.eclipse.equinox.executable.feature.group" version="0.0.0"/>
+            <unit id="org.eclipse.platform.feature.group" version="0.0.0"/>
             <unit id="org.eclipse.equinox.p2.core.feature.source.feature.group" version="0.0.0"/>
             <unit id="org.eclipse.equinox.sdk.feature.group" version="0.0.0"/>
-            <unit id="org.eclipse.jdt.source.feature.group" version="0.0.0"/>
-            <unit id="org.eclipse.sdk.feature.group" version="0.0.0"/>
             <unit id="org.mockito.mockito-core" version="0.0.0"/>
             <unit id="org.apache.commons.commons-io" version="0.0.0"/>
+            <unit id="org.eclipse.jdt.apt.pluggable.core" version="0.0.0" />
```

Comment:
Is there a reason why `jdt.apt.pluggable.core` isn't being built by the incubator at https://github.com/eclipse-jdtls/eclipse-jdt-core-incubator/blob/develop/repository/category.xml ? Either way it's probably a good idea to make sure the incubator produces a snapshot build that matches the I-build from eclipse/updates.

Reply:
It build it but doesn't publish it . Do you want to publish it to downloads too? As I don't see any changes needed for any apt bundles I would even say it might be better to even not publish jdt.apt.core but use it from the I-build repo.